packages/react-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@panderalabs/cra-react-redux-scripts",
-  "version": "1.2.4",
+  "version": "2.0.0",
```

Comment:
I updated to version `2.0.0` since some of the dependencies were updated to the next major release.  This may not be necessary.  However it would help to test this on any existing projects that use this library.

Reply:
:+1: